src/main/java/Deadline.java

```diff
@@ -0,0 +1,25 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    public Deadline(String description){
+        super(description);
+    }
+    
+    public Deadline(String description, LocalDate date) {
+        super(description, date);
+    }
+
+    public Deadline(String description, LocalDate date, String duration) {
+        super(description, date, duration);
+    }
```

Comment:
I like the functionality of letting the user choose whether to input a duration 👍